js/src/modals/AddAddress/addAddress.js

```diff
@@ -28,6 +28,7 @@ export default class AddAddress extends Component {
 
   static propTypes = {
     contacts: PropTypes.object.isRequired,
+    address: PropTypes.string,
```

Comment:
Well, we don't conform to that at all currently. With the Redux issues and not being able to pass required props this whole methodology falls apart. (e.g. see isTest which is absolutely required and just currently needs to be moved to non-required since it just errors all-over)

So there are 3 categories, Required, non-required & Redux. It just becomes an absolute mess and mental overhead to manage.